openfisca_tasks/test_code.mk

```
@@ -10,7 +10,7 @@ install: install-deps install-edit install-test
 ## Enable regression testing with template repositories.
 install-test:
 	@$(call print_help,$@:)
-	@pip install --upgrade --no-dependencies openfisca-country-template
+	@pip install git+https://github.com/openfisca/country-template@training-upgrade-fix
```

v7 [has been published](https://pypi.org/project/openfisca-country-template/) and should integrate the required fix via 49726ba24fcd61c6d1a0629077ff98d997f96132 🙂 
```suggestion
	@pip install --upgrade --no-dependencies openfisca-country-template
```